server/gateway/sample.env

```diff
@@ -0,0 +1,5 @@
+MICROSOFT_CONFIGURATION_CLIENT_ID=
```

Comment:
Can you add some information about the two different id/secret parings. Either inline or to the readme

Reply:
I remembered why but can't remember anymore :( @SamBroner why do we have linked accounts in gateway

Reply:
Added the steps to readme to add secrets. Lmk if there is anything else I should add there

Reply:
We had linked accounts to test out our host scopes